arangod/Aql/Aggregator.cpp

```diff
@@ -183,12 +182,11 @@ struct AggregatorMin final : public Aggregator {
     }
   }
 
-  AqlValue stealValue() override {
+  AqlValue get() const override {
     if (value.isEmpty()) {
       return AqlValue(AqlValueHintNull());
     }
     AqlValue copy = value;
-    value.erase();
     return copy;
```

Comment:
It looks like the ownership for the resulting values is unclear now, as some of the `get` methods return a reference to the data (e.g. AggregatorMin and AggregatorMax), but others are creating a full copy (e.g. AggregatorAverageStep1, AggregatorUnique).

Reply:
It looks like that the caller in WindowExecutor.cpp assumes ownership of the values returned from here, so for Min/Max we need to create copies as well.
The result of Min/Max is not necessarily a number, but it can be of arbitrary type (e.g. string), so we must copy here.

Reply:
there is no semantic difference, from now vs previously

Reply:
The previous implementation handed over ownership. The current version doesn't hand over ownership here.

Reply:
The `erase` call in the previous implementation meant that the Aggregator gave up the ownership. Now it doesn't.

Reply:
exactly, it is not suppoed to give up ownership